sim/common/sod/item_effects/phase_5.go

```diff
@@ -161,25 +185,122 @@ func init() {
 		})
 	})
 
-	core.AddEffectsToTest = true
-}
+	// https://www.wowhead.com/classic/item=231784/lightnings-cell
+	//
 
-	// https://www.wowhead.com/classic/spell=465414/crusaders-zeal
-	// Used by:
-	// - https://www.wowhead.com/classic/item=229806/truthbearer and
-	// - https://www.wowhead.com/classic/item=229749/truthbearer
-	func crusadersZealAura465414(character *core.Character) *core.Aura {
-		return character.GetOrRegisterAura(core.Aura{
-			ActionID: core.ActionID{SpellID: 465414},
-			Label:    "Crusaders Zeal (465414)",
-			Duration: time.Second * 8,
+	// Use: Energizes a Paladin with light, increasing melee attack speed by 25%
+	// and spell casting speed by 33% for 20 sec. (2 Min Cooldown)
+	core.NewItemEffect(ScrollsOfBlindingLight, func(agent core.Agent) {
```

Comment:
Could you also add this one to the paladin too?

Reply:
done